test/parallel/test-fs-open-flags.js

```diff
@@ -21,9 +21,8 @@
 
 // Flags: --expose_internals
 'use strict';
-require('../common');
 const assert = require('assert');
-
+const common = require('../common');
```

Comment:
Shouldn't `common` be required before `assert`?

Reply:
ah right... good catch

Reply:
Done.